tests/suite/zar/s3/import.yaml

```diff
@@ -0,0 +1,51 @@
+script: |
+  export AWS_REGION=us-east-2 AWS_ACCESS_KEY_ID=minioadmin \
+        AWS_SECRET_ACCESS_KEY=minioadmin AWS_S3_ENDPOINT=http://localhost:9000
+  mkdir -p data/bucket
+  minio server --quiet --address "localhost:9000" ./data &> /dev/null &
+  pid=$!
+  until nc -vzw 2 localhost 9000; do sleep 2; done
+  zar import -s 2500B -R ./root -data s3://bucket/zartest babble.tzng
```

Comment:
Consider using `-s 20KiB`, as it will reduce the number of zng logs to just 2.

Reply:
good idea.